stats/container.go

```diff
@@ -29,7 +26,7 @@ func initContainerManager() {
 		return
 	}
 	// TODO: support influxdb or other backend storage
-	cmgr, err = manager.New(memory.New(storageDuration, nil), sysFs, maxHousekeepingInterval, allowDynamicHousekeeping)
+	cmgr, err = manager.New(memory.New(storageDuration, nil), sysFs)
```

Comment:
could you update to latest github.com/google/cadvisor ? then the original code will work.